packages/salesforcedx-vscode-replay-debugger/src/index.ts

```diff
@@ -172,23 +183,36 @@ export async function retrieveLineBreakpointInfo(): Promise<boolean> {
         breakpointUtil.createMappingsFromLineBreakpointInfo(lineBpInfo);
         return true;
       } else {
-        vscode.window.showErrorMessage(
-          nls.localize('no_line_breakpoint_information_for_current_project')
+        const errorMessage = nls.localize(
+          'no_line_breakpoint_information_for_current_project'
         );
+        writeToDebuggerOutputWindow(errorMessage);
+        vscode.window.showErrorMessage(errorMessage);
         return true;
       }
     }
   } else {
-    vscode.window.showErrorMessage(
-      nls.localize('session_language_server_error_text')
-    );
+    const errorMessage = nls.localize('session_language_server_error_text');
+    writeToDebuggerOutputWindow(errorMessage);
+    vscode.window.showErrorMessage(errorMessage);
     return false;
   }
 }
 
 function imposeSlightDelay(ms = 0) {
   return new Promise(r => setTimeout(r, ms));
 }
+
+const sfdxCoreExtension = vscode.extensions.getExtension(
+  'salesforce.salesforcedx-vscode-core'
```

Comment:
You had questions about the core extension not being loaded before replay debugger extension. Are those concerns still valid? Does this have to be done elsewhere?

Reply:
I had to change the way it was used, hence the way this was written. The pattern that other extensions are using is to declare the following:
const sfdxCoreExports = vscode.extensions.getExtension(
  'salesforce.salesforcedx-vscode-core'
)!.exports;
const channelService = sfdxCoreExports.channelService;
Notice the !.exports? I was originally doing that but it was causing problem that were surfacing when trying to run extension tests.